src/themis/secure_comparator.c

```diff
@@ -169,7 +171,7 @@ static themis_status_t ed_verify(uint8_t pos, const ge_p3 *point, const uint8_t
 
 	sc_reduce(k);
 
-	if (memcmp(k, signature, ED25519_GE_LENGTH))
+	if (cst_time_memcmp(k, signature, ED25519_GE_LENGTH))
```

Comment:
Not sure this is needed: this is signature verification and signature is public here.